sales_chr24_photoedit_user/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020  Timeware Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale group for Chrono24 and Products",
+    "category": "Security",
+    "version": "12.0.0.1",
+    "author": "Timeware limited",
```

Comment:
Quartile Limited, Timeware Limited
Or you change all to Timeware Limited is also fine, but please this consistent within module